Source/Core/Core/FloatUtils.h

```diff
@@ -0,0 +1,191 @@
+// Copyright 2018 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include <array>
+#include <bit>
+#include <limits>
+
+#include "Common/CommonTypes.h"
+#include "Core/PowerPC/Gekko.h"
+
+namespace Common
```

Comment:
This file is no longer in the Common directory, so it shouldn't be using the Common namespace.